tools/buildsys/src/builder.rs

```diff
@@ -464,8 +557,12 @@ fn docker(args: &[String], retry: Retry) -> Result<Output> {
             .run()
             .context(error::CommandStartSnafu)?;
 
+        // Print `docker` output to stdout for non-cleanup tasks.
         let stdout = String::from_utf8_lossy(&output.stdout);
-        println!("{}", &stdout);
+        if !args[0].starts_with("rm") {
+            println!("{}", &stdout);
+        }
```

Comment:
I could regex the output if we want to suppress the specific error I'm trying to avoid, but I thought we might not actually care about the output from `rm` or `rmi`.

```
Error response from daemon: No such container: buildsys-repack-aws-k8s-1.27-aarch64-b4041b87905a

Error response from daemon: No such image: buildsys-repack-aws-k8s-1.27-aarch64-b4041b87905a:latest
```